src/HealthGPS.Tests/HealthGPS.Test.cpp

```diff
@@ -17,7 +17,7 @@ namespace fs = std::filesystem;
 
 static std::vector<hgps::MappingEntry> create_mapping_entries() {
     using namespace hgps;
-    return std::vector<MappingEntry>{MappingEntry("Year", 0, core::Identifier::empty(), true),
+    return std::vector<MappingEntry>{MappingEntry("Year", 0, core::Identifier::empty()),
```

Comment:
I think the compiler can infer the type from the return type here:

```suggestion
    return {MappingEntry("Year", 0, core::Identifier::empty()),
```

(You'll need to `clang-format` this.)